components/eam/src/chemistry/mozart/mo_exp_sol.F90

```diff
@@ -109,11 +109,18 @@ subroutine exp_sol( base_sol, reaction_rates, het_rates, extfrc, delt, xhnm, nco
                 base_sol(i,k,l) = base_sol(i,k,l)*exp(-delt*loss(i,k,m)/base_sol(i,k,l)) + delt*(prod(i,k,m)+ind_prd(i,k,m))
              end do
           end do
-       ! apply brc -> pom decay to entire atmosphere
+       ! apply brc -> pom decay to troposphere
+       ! apply ext emission to whole atmosphere in case of large fires
        elseif (trim(solsym(l)) == 'brc_a4' .or. trim(solsym(l)) == 'brc_a1' .or. trim(solsym(l)) == 'brc_a3') then
                ! V2-like explicit equation 
           do i = 1,ncol
-             do k = 1,pver
+             do k = 1,ltrop(i)
+             ! above tropopause, emission only
+                chem_loss(i,k,l) = 0.0_r8
+                chem_prod(i,k,l) = ind_prd(i,k,m) 
+                base_sol(i,k,l) = base_sol(i,k,l) + delt * ind_prd(i,k,m)  
+             end do               
```

Comment:
@keziming , can you confirm by a short run that without this part of code, the 3D emissions above the tropopause will not be applied? If confirmed, what about applying the e-folding decay everywhere? The rationales are 1) simplify code; 2) is there a strong reason why BrC doesn't decay in the stratosphere?